### DISCUSSION TITLE
How to add a custom css class

### DISCUSSION THREAD

**Discussion by fablexis** [vueform/vueform#160] [Questions] [answered]
I'm trying to add a custom css class to my form using the `class` key as part of my object for my schema (I also tried using `addClass` just like the example of the docs), but despite I can see that class as part of my element in the DOM, I can't see the styles applied of that css class

This is what I'm doing

```
<template>
  <v-container>
    <Vueform :schema="schema"></Vueform>
  </v-container>
</template>

<script setup lang="ts">
const schema = {
  firstList: {
    type: 'list',
    sort: true,
    class: 'my-custom-class', //here I have my css class reference
    controls: {
      add: false,
      remove: true
    },
    element: {
      type: 'object',
      schema: {
        simpleText: {
          type: 'text',
          label: '1st this is a simple input text in a list'
        },
        datepicker: {
          type: 'date',
          label: '1st this is a datepicker'
        }
      }
    }
  },
  secondList: {
    type: 'list',
    sort: true,
    controls: {
      add: false,
      remove: true
    },
    element: {
      type: 'object',
      schema: {
        simpleText: {
          type: 'text',
          label: '2nd this is a simple input text in a list'
        },
        datepicker: {
          type: 'date',
          label: '2nd this is a datepicker'
        }
      }
    }
  }
}
</script>

<style lang="scss" scoped>
.my-custom-class {
  border: 1px solid #546122;
  margin: 0;
  outline: 0;
  cursor: pointer;
  border-radius: 4px;
  padding: 16px 42px 16px 26px;
  background: #fff;
}
</style>
```

When I inspect the element, I see the `my-custom-class` as part of the class attr, but there's nothing about its styles

![image](https://github.com/vueform/vueform/assets/7627658/c646ea0e-3d16-437b-b06d-0a2501d34cfa)

So, my question is: how can I add a custom class for any element using this object notation for the schema?

@adamberecz

**Comment by nclemeur** [comment]
Did you try without the "scoped" in the style?

**Comment by fablexis** [comment, marked as answer]
I realized that the `scoped` attr prevents applying custom css... as soon as I removed `scoped` from my `<stlyle>`tag, my custom styles worked

**Comment by adamberecz** [comment]
Also, you can read this section about modifying classes beyond adding a `class` to top level DOMs:
https://vueform.com/docs/styles-and-layout#changing-classes
https://vueform.com/docs/styles-and-layout#add-classes
